index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap" rel="stylesheet">
+  <title>Portfolio Sneak Peak</title>
+</head>
+<body>
+  <div class="container">
+    <header>
+      <img class="status-bar" src="images/status-bar.png" alt="Portfolio Status Bar">
+      <nav>
+        <div class="logo">
+          <img src="images/app-bar.png" alt="Logo image">
+        </div>
+      </nav>
+    </header>
+    <section>
+      <h1>I’m Sylvester,
+          Glad to see you!
+      </h1>
+    </section>
+    <section>
+      <p class="intro">
+        I’m a software developer! I can help you build a product, 
+        feature or website. Look through some of my work and experience! 
+        If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <h3>LET'S CONNECT</h3>
```

Comment:
- [ ] Kindly refer to the design template and make sure you are using the same design in the template, look out for:
- font size, color.
- [ ] The button (Let's Connect) is a link `a` or `button` tag, and follow the button styling: Active, Hover and Pressed.
![image](https://user-images.githubusercontent.com/49621279/120674997-85b7bd00-c449-11eb-9db8-a2847f5686f5.png)

Reply:
## These reviews are helpful. 👏 👏 

**However, from what I'm looking at, I cant seem to pick any hover or click styling(s).**

Maybe this is from my end

Reply:
Hi @Sly-Cloud , thanks for your feedback, I've updated the review to better communicate what I meant. It essentially means checking in with the design to see what features are supposed to be present in that element. I hope it helps now!